geode-core/src/test/java/org/apache/geode/internal/cache/partitioned/PartitionedRegionRebalanceOpTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.cache.partitioned;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Map;
+
+import org.junit.Test;
+
+import org.apache.geode.cache.PartitionAttributes;
+import org.apache.geode.cache.PartitionAttributesFactory;
+import org.apache.geode.internal.cache.InternalCache;
+import org.apache.geode.internal.cache.PartitionedRegion;
+import org.apache.geode.internal.cache.PartitionedRegionTestHelper;
+import org.apache.geode.internal.cache.control.InternalResourceManager;
+import org.apache.geode.internal.cache.partitioned.rebalance.model.Member;
+import org.apache.geode.internal.cache.partitioned.rebalance.model.PartitionedRegionLoadModel;
+
+
+public class PartitionedRegionRebalanceOpTest {
+  private PartitionedRegion leaderRegion;
+  private PartitionedRegion colocRegion1;
+  private PartitionedRegion colocRegion2;
+  private static final long MB = 1024 * 1024;
+
+  @Test
+  public void testAllCollocatedRegionsSetWhenRebalanceTargetIsLeaderRegion() {
+    leaderRegion = (PartitionedRegion) PartitionedRegionTestHelper.createPartionedRegion("leader");
+    PartitionAttributesFactory paf = new PartitionAttributesFactory();
+    PartitionAttributes pa = paf.setColocatedWith("/leader").create();
+    colocRegion1 =
+        (PartitionedRegion) PartitionedRegionTestHelper.createPartionedRegion("colo1", pa);
+    colocRegion2 =
+        (PartitionedRegion) PartitionedRegionTestHelper.createPartionedRegion("colo2", pa);
+
+    try {
+      PartitionedRegionRebalanceOp part_op =
+          new PartitionedRegionRebalanceOp(leaderRegion, false, null, false, false, null, null);
+      part_op.checkAndSetColocatedRegions();
+      assertEquals(3, part_op.colocatedRegions.size());
```

Comment:
Please replace all with AssertJ.

Reply:
Hi Mario,
I'm changing the test case since Alberto also commented it's better not to change existing methods from private to protected. I'm checking whether it is OK to add a distributed test case, will take into account AssertJ also, thanks.